script.js

```diff
@@ -6,4 +9,317 @@ function openNav(){
 //closing navbar
 function closeNav(){
 document.getElementById("mySidenav").style.width="0";
-}
\ No newline at end of file
+}
+
+
+
+
+//***********season config js***************//
+
+//adds new season from input box
+function addNewSeason()
+{
+    var newSeason = $("#newSeasonInputBox").val().trim().toString();
+
+    var isAlreadyStored = false;
+    if (document.cookie.length > 0) //if cookies are stored at all
+    {
+        if (getCookie("seasonList") !== " ")
+        {
+            var seasonNames = getCookie("seasonList").split(",");
+            for (var i=0;i<seasonNames.length;i++)
+            {
+                var val = seasonNames[i].trim();
+                if (newSeason==val.toString())
+                {
+                    isAlreadyStored = true;
+                    break;
+                }
+            }
+        }
+    }
+
+    if (isAlreadyStored) //if season is already stored
+    {
+        $("#newSeasonConfirmation").html("<i>Season \'" + newSeason +"\' already exists.</i>");    
+    } else
+    {
+        //add to radioList
+        $("#newSeasonConfirmation").html("<i>Season " + "\'"+ newSeason +"\'"+" has been added.</i>"); //confirmation text
+        $("#seasonRadioList").append("<label><input type=\"radio\" name=\"seasonListItem\" value=\"" + newSeason + "\">" + newSeason + "</label><br>"); //add to radiolist
+
+        //save newSeason in cookie
+        document.cookie = "seasonList=" + getCookie("seasonList") + newSeason + "," + "; expires=Wed, 1 Jan 2038 12:00:00 UTC; path=/";
+
+        $("#newSeasonInputBox").val("");
+    }
+}
+
+//meant for only the startup of the page
+function loadSeasons()
+{
+    if (document.cookie.length > 0)
+    {
+        if (getCookie("seasonList") !== " ")
+        {
+            var seasonList = getCookie('seasonList').split(",");
+            //console.log('seasonList:' + seasonList);
+            for (var i=0;i<seasonList.length-1;i++) //length-1 because last will always be "" because name,name,
+            {
+                var cookieVal = seasonList[i];
+                $("#seasonRadioList").append("<label><input type=\"radio\" name=\"seasonListItem\" value=\"" +cookieVal + "\">"+cookieVal+ "</label><br>");
+            }
+        }
+    }
+
```

Comment:
If you're going to use the cookie to determine what season is selected each time you go to save (like at [line 291](https://github.com/Team6321/Scouting_App/pull/1/files#diff-9a9569e9d73f33740eada95275da7f30R291)), you should select this based on that cookie here as well - so the app doesn't behave like a season is selected when it doesn't look like one is.

That would look like (after you've loaded the curentSeason):

```JS
$(`input[name=seasonListItem][value="${ currentSeason }"]`).attr('checked', true);
```

`.attr(` gives access to just about any attribute on an element (text, html, value, checked) and is the under-the-hood code for most if not all of the easy accessors (`.text(`) - we use it when accessing (or setting) an attribute on an object that only that type has (hence, low return on investment in making an easy accessor for it).